test-integ/topoutil/actions.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
What do you think about having all these `topoutil` things in `sprawl(test)` or `deployer` itself? I considered it but chickened out. For example, you could have `(*topology.Config).DisableNode(...)`.

Reply:
I think I can move the enable/disable stuff and the relationship stuff, but Asserter and the fixtures probably have to stay for now.

Reply:
moved

Reply:
> but Asserter and the fixtures probably have to stay for now.
Why's that? 

I'm reluctant to have yet another `util` package. And it's weird to have a shared lib inside integration tests IMO.